cmd/kube-proxy/app/server.go

```diff
@@ -22,6 +22,7 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/client-go/metadata"
```

Comment:
Group this with other k8s.io imports please

Reply:
done